accept-leads-default.md

```
@@ -1,38 +1,63 @@
-## Posting Guidelines
+# Posting Guidelines
 
-This document provides instructions on implementing a feed that may be used to immediately notify your CRM system of a lead. These values are sent under the POST method. The following information outlines the format of the data that will be sent to you.
+This document provides instructions on implementing a feed that may be used to immediately notify your CRM system of a lead. Values are sent under the POST method. The following outlines the format of the data that will be sent to you.
 
-You may receive more fields than listed below in a POST; however any field not listed here is deprecated.
+You may receive more fields than listed below in a POST;
 
 ### Possible Attributes and Values
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)